src/backends/oracle/statement.cpp

```diff
@@ -164,21 +164,29 @@ std::string oracle_statement_backend::rewrite_for_procedure_call(
 
 int oracle_statement_backend::prepare_for_describe()
 {
-    sword res = OCIStmtExecute(session_.svchp_, stmtp_, session_.errhp_,
-        1, 0, 0, 0, OCI_DESCRIBE_ONLY);
-    if (res != OCI_SUCCESS)
-    {
-        throw_oracle_soci_error(res, session_.errhp_);
-    }
+    ub2 statementType = 0;
+    sword res = OCIAttrGet(static_cast<dvoid*>(stmtp_),
```

Comment:
Shouldn't we do something with `res`? I am not actually sure what should happen if an error happens here, but it looks like we ought to handle it somehow.

Reply:
Yes I agree, we should throw throw_oracle_soci_error here if an error occurs.